content/posts/supports/index.md

```diff
@@ -9,14 +9,14 @@ menu:
     weight: 660
 ---
 
-This theme supports adding various support/donation links in your site. Currently, supported support links are:
+Ce thème supporte l'ajout de liens de soutien/donation sur votre site. Actuellement, les liens de soutien supportés sont: 
```

Comment:
I suspect you have copy pasted the changes from you French version. Can you make the changes in English?

Reply:
You're right too. Yes, i make change in English !